perfmetrics/scripts/run_load_test_and_fetch_metrics.sh

```diff
@@ -35,7 +35,7 @@ echo "Mounting gcs bucket"
 mkdir -p gcs
 GCSFUSE_FLAGS=$1
 UPLOAD_FLAGS=$2
-BUCKET_NAME=periodic-perf-tests
+BUCKET_NAME="periodic-perf-tests${EXPERIMENT_NUMBER}"
```

Comment:
What happens when we set `EXPERIMENT_NUMBER` with empty value?

Can we add a check to fail the job if the environment variable is set but empty, just to make sure to not set this environment variable with empty value?

Reply:
Or if we don't then please pass the BucketName too as a command line arguments.

Reply:
`Can we add a check to fail the job if the environment variable is set but empty, just to make sure to not set this environment variable with empty value?` For [this](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh) tests `EXPERIMENT_NUMBER` is unset(empty value). when we will remove this setup at that time we can add this condition I guess.